### PR TITLE
(documentation) volume opions for config file

### PR DESCRIPTION
**Pull request by PierreKircher**
that makes the in container edits obsolet and the config file can be mapped in from a distributed fs to example

### CODE REVIEW

**Comment by thibaultCha** [comment]
Nice. The docs could use rewording tho: "You can load a configuration file from your local file system"

And for the example:

```
... -v <configuration file path>:/etc/kong
```

As we always refer like that to a configuration file in our docs.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ Once Cassandra is running, we can start the Kong container and link it with the
 docker run -p 8000:8000 -p 8001:8001 -d --name kong --link cassandra:cassandra mashape/docker-kong:0.2.0-2
 ```
 
+to pass in a propper configfile without the need to docker exec and reload the container use
```
Comment (Nijikokun) [inline README.md]: Can you change this line to be:

> To reload the configuration file without accessing the docker shell:

**Comment by PierreKircher** [comment]
done

**Comment by ahmadnassri** [comment]
@PierreKircher awesome, thank you!

**Pull request merged**